### PR TITLE
Doxygen in INITPOST.F, INITPOST_GFS_NEMS_MPIIO, INITPOST_NEMS.f and INITPOST_NETCDF.f.

### PR DESCRIPTION
**Pull request by kayeekayee**
This is part of Issue #392.

Fixes the doxygen warnings in INITPOST.F, INITPOST_GFS_NEMS_MPIIO, INITPOST_NEMS.f and INITPOST_NETCDF.f.

Please review the changes. Thank you.

### CODE REVIEW

**Comment by kayeekayee** [comment]
@edwardhartnett Would you please review this PR?

File: sorc/ncep_post.fd/INITPOST.F [hunk 1]
```diff
@@ -1,46 +1,24 @@
...
-!!       COMMON   - CTLBLK
-!!                  LOOKUP
-!!                  SOILDEPTH
-!!
-!!    
-!!   ATTRIBUTES:
-!!     LANGUAGE: FORTRAN
-!!     MACHINE : CRAY C-90
-!!
+!> @brief initpost() initializes post for run.
```
Comment (edwardhartnett) [inline sorc/ncep_post.fd/INITPOST.F:2]: There needs to be two doxygen blocks, one for the file, one for the subprogram.

File: sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f [hunk 2]
```diff
@@ -1,51 +1,29 @@
...
-!!       COMMON   - CTLBLK
-!!                  LOOKUP
-!!                  SOILDEPTH
-!!
-!!    
-!!   ATTRIBUTES:
-!!     LANGUAGE: FORTRAN
-!!     MACHINE : CRAY C-90
-!!
+!> @brief initpost_gfs_nems_mpiio() initializes post for run.
```
Comment (edwardhartnett) [inline sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f:2]: Two doxygen blocks are needed...

**Review by edwardhartnett — CHANGES_REQUESTED** [review]

File: sorc/ncep_post.fd/INITPOST.F (hunk 1 above)
Comment (kayeekayee) [reply sorc/ncep_post.fd/INITPOST.F:2]: Thank you so much for reviewing, Ed! I updated them all. Please have a look at them.

File: sorc/ncep_post.fd/INITPOST_NETCDF.f [hunk 3]
```diff
@@ -1,6 +1,8 @@
 !> @file
-!> @brief initpost_netcdf() initializes post for run.
+!> !brief initpost_netcdf() initializes post for run.
```
Comment (edwardhartnett) [inline sorc/ncep_post.fd/INITPOST_NETCDF.f]: Messed up the brief tag here...

Comment (kayeekayee) [reply sorc/ncep_post.fd/INITPOST_NETCDF.f]: Good catch!

**Review by edwardhartnett — APPROVED** [review]

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were done on Hera. No changed results were found.

**Pull request merged**